src/search/search.component.spec.ts

```diff
@@ -23,7 +23,7 @@ describe("Search", () => {
 			],
 			providers: []
 		})
-		.compileComponents();
+			.compileComponents();
```

Comment:
```suggestion
		.compileComponents();
```

Looks cleaner to keep it aligned?

Reply:
It's possible to allow this, but then I need to turn of the `indent` rule for MemberExpressions. 

This however can lead to situations like this:
<img width="1238" alt="Screenshot 2025-02-26 at 14 40 53" src="https://github.com/user-attachments/assets/55b10376-6583-466c-bb40-e8d20a81cbbc" />